index.html

```diff
@@ -5663,7 +5663,21 @@ <h3>Language and Base Direction</h3>
   "@id": "https://www.w3.org/2018/credentials/examples#title"
 }
         </pre>
+      </section>
+
+      <section class="substantive">
+        <h3>Providing Default Language and Direction</h3>
 
+        <p>
+A default language and direction MAY be provided for the entire credential.
+However, doing so introduces a higher demand on context creators and 
+downstream processors. Consequently, using default language and
+direction is expected to be be limited to scenarios where JSON-LD expansion-based
+transformation can be assured. See the
+<a date-cite="JSON-LD#string-internationalization">String
+Internationalization</a> section of the [[JSON-LD]] specification for more
+information.
```

Comment:
I don't agree that this text fixes the problem, as it is still "MAY" with harsh sounding warning afterwards. Perhaps:

```suggestion
The language and direction of each natural language string field MUST be provided 
via the language value structure or via a default language and direction
for the entire credential. Using the language value structure is preferred because document
defaults can result in the need to perform JSON-LD expansion-based transformation by
downstream processors. See the <a data-cite="JSON-LD#string-internationalization">String
Internationalization</a> section of the [[JSON-LD]] specification for more information.

Reply:
@aphillips I'm not sure we can make this a MUST as it would effectively force all v2 Verifiable Credentials to contain language metadata which they may not need/use/etc. It's a bit analogous to if `lang` on `<html>` were to suddenly become a requirement for pages to render.

I'd prefer all data come with proper language metadata...but I'm not sure this is the layer at which to force it into existence.

Reply:
@jandrieu,

> I agree with SHOULD on this

Well, if we change it from MAY to SHOULD, then we need it to be you SHOULD use the default language if and only if you don't use the localized language objects (as those are preferred). So we'd need some additional tweaks, we can't just change MAY => SHOULD here.

Reply:
Since this is 2.0, I can understand making MUST => SHOULD in my suggested text as a sop to existing credentials that do not include language metadata. 

I would clarify that this part of @BigBlueHat's comment:

> as it would effectively force all v2 Verifiable Credentials to contain language metadata which they may not need/use/etc.

I don't care if "they" use the metadata. Some consumers are just passing the data or checking the validity dates, after all. If the right credential was received, the display name isn't important. But I do care that the metadata is present for when consumers do access the natural language strings. That's not really a "SHOULD" thing.

What should be added if we make this a SHOULD is some additional text at the end:

```suggestion
The language and direction of each natural language string field SHOULD be provided 
via the language value structure or via a default language and direction
for the entire credential. Using the language value structure is preferred because document
defaults can result in the need to perform JSON-LD expansion-based transformation by
downstream processors. See the <a data-cite="JSON-LD#string-internationalization">String
Internationalization</a> section of the [[JSON-LD]] specification for more information.
Natural language string values that do not have a language associated with them
SHOULD treat the language as Undefined (language tag `und`). Natural language string values
that do not have a base paragraph direction associated with them SHOULD treat the value
as having a direction of "auto".
```

Reply:
+1 to @aphillips suggestion.

Reply:
Excellent, thank you @aphillips -- I believe that will achieve consensus given the discussion the group had yesterday.

Reply:
Tweaks to @aphillips [suggestion](https://github.com/w3c/vc-data-model/pull/1339/files#r1393063076)

```suggestion
The language and base direction of each natural language string property SHOULD
be provided, either via the language value structure for each property or via a
default language and base direction for the entire credential. Using the
language value structure is preferred, because document defaults can result in
a requirement that downstream processors perform JSON-LD expansion-based
transformation which is otherwise optional. See the <a
data-cite="JSON-LD#string-internationalization">String Internationalization</a>
section of the [[JSON-LD]] specification for more information. Natural language
string values that do not have a language associated with them SHOULD be
treated as if the language value is `Undefined` (language tag "`und`"). Natural
language string values that do not have a base direction associated with them
SHOULD be treated as if the direction value is "`auto`".
```